nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -871,7 +871,7 @@ $(XamlMetaDataProviderPch)
     <ItemDefinitionGroup>
         <ClCompile>
             <AdditionalOptions>%(AdditionalOptions) /bigobj</AdditionalOptions>
-            <AdditionalOptions Condition="'%(ClCompile.LanguageStandard)' == 'stdcpp17'">%(AdditionalOptions) /await</AdditionalOptions>
+            <AdditionalOptions Condition="'%(ClCompile.LanguageStandard)' == 'stdcpp17' And '$(CppWinRTEnableLegacyCoroutines)' != 'false'">%(AdditionalOptions) /await</AdditionalOptions>
```

Comment:
Why not just remove this, or does this affect existing projects?

Reply:
Removing it entirely could affect projects which relied on the implicit addition of /await.

Reply:
Makes sense, the logic is just throwing me but then msbuild has never made sense to me.